testdata/scripts/seed.txt

```diff
@@ -0,0 +1,36 @@
+# Check the binary with a given base64 encoded seed
+garble -debugdir=./debug -literals -seed=OQg9kACEECQ= build main.go
+exec ./main
+cmp stdout main.stdout
+grep \\x42\\x49\\x29\\x2f\\x5a\\x13\\xab\\xef\\xfa\\x26\\x33\\x3e\\x46\\xa9\\xec\\x3e\\x96\\xa3\\x73\\x15\\x7c\\xa2\\x38\\xf2\\xf0\\xaf\\x58\\x6a\\xac\\x2b\\x50\\x94\\x3b\\x5b\\x3e\\xda\\x2f\\xe7 $WORK/debug/main/z0.go
```

Comment:
you don't need to use $WORK. you can just use relative paths like debug/etc

Reply:
OK